modules/core/02-client/types/keys.go

```diff
@@ -25,6 +25,9 @@ const (
 	// KeyNextClientSequence is the key used to store the next client sequence in
 	// the keeper.
 	KeyNextClientSequence = "nextClientSequence"
+
+	// ParamsKey is the store key for the IBC client parameters
+	ParamsKey = "clientParams"
```

Comment:
nit: in other PRs, this is called `"params"`. So maybe we should keep doing this?

Reply:
IBC submodules share the store space (all use the core IBC store), so this would cause collisions with connection params etc